docs/resources/scim_integration.md

```diff
@@ -12,11 +12,14 @@ description: |-
 ## Example Usage
 
 ```terraform
-resource "snowflake_scim_integration" "aad" {
-  name             = "AAD_PROVISIONING"
-  network_policy   = "AAD_NETWORK_POLICY"
-  provisioner_role = "AAD_PROVISIONER"
-  scim_client      = "AZURE"
+resource "snowflake_scim_integration" "test" {
```

Comment:
let's have a minimal example and all fields example

Reply:
Done.